dedupe/deduper.py

```diff
@@ -149,18 +150,6 @@ def remove_duplicate(self, duplicate_package, retained_package):
 
         self.ckan_api.remove_package(duplicate_package['id'])
 
-        if(len(duplicate_package['name']) < len(retained_package['name'])
-            and self.update_name ):
-            # If the package to be retained has extra random character at
-            #  the end of the name, we want to rename it to the "standard"
-            #  name to keep the typical URL. 
-            self.log.info('Renaming kept package from %s to %s',
-                      retained_package['name'], duplicate_package['name'])
-            retained_package['name'] = duplicate_package['name']
-            self.ckan_api.update_package(retained_package)
-            if self.removed_package_log:
-                self.removed_package_log.add(retained_package)
-
```

Comment:
You didn't grab the latest main branch, otherwise this removal wouldn't be in here. Try git rebase, or manually inspect the latest [PR merge](https://github.com/GSA/datagov-dedupe/pull/21/files) and pull in the changes. If you fix this, I believe it will fix the failing test...